templates/news/news-en.html

```diff
@@ -3,224 +3,484 @@
+							<option>job seekers</option>
+							<option>media</option>
+							<option>non-Canadians</option>
+							<option>non-governmental organizations</option>
+							<option>parents</option>
+							<option>persons with disabilities</option>
+							<option>rural community</option>
+							<option>scientists</option>
+							<option>seniors</option>
+							<option>students</option>
+							<option>travellers</option>
+							<option>veterans</option>
+							<option>visitors to Canada</option>
+							<option>women</option>
+							<option>youth</option>
+						</select>
+					</div>
+					<div class="form-group">
+						<label for="dt_location">Region</label>
+						<select class="form-control" id="dt_location" name="dt_location" data-column="5">
+							<option value="">Any</option>
+							<option>Atlantic</option>
+							<option>British Columbia</option>
+							<option>Ontario</option>
+							<option>Prairies</option>
+							<option>Quebec</option>
+							<option>Territories</option>
+						</select>
+					</div>
+					<div class="row">
+						<div class="col-xs-6">
+							<button type="submit" class="btn btn-primary" aria-controls="dataset-filter">Filter</button>
+						</div>
+						<div class="col-xs-6">
+							<button type="reset" class="btn btn-default">Reset filters</button>
+						</div>
+					</div>
+				</form>
+			</details>
+		</section>
+		<section>
 			<h2 class="h4">Resources</h2>
-			<ul>
-				<li><a href="#">Canada Gazette</a></li>
-				<li><a href="#">Hansard</a></li>
-				<li><a href="#">Media Contacts</a></li>
+			<ul class="list-group">
+				<li class="list-group-item"><a href="#">Canada Gazette</a></li>
+				<li class="list-group-item"><a href="#">Hansard</a></li>
+				<li class="list-group-item"><a href="#">Media Contacts</a></li>
 			</ul>
 		</section>
 	</div>
 	<div class="col-md-9">
-		<table class="wb-tables table nws-tbl" id="dataset-filter" aria-live="polite" data-wb-tables='{
-		"bDeferRender": true,
-		"ajaxSource": "https://wet-boew.github.io/wet-boew/demos/tables/ajax/datasource.json",
-		"order": [1, "desc"],
-		"columns": [
-			{ "data": "TITLE", "className": "nws-tbl-ttl h4" },
-			{ "data": "PUBDATE", "className": "nws-tbl-date" },
-			{ "data": "DEPT", "className": "nws-tbl-dept" },
-			{ "data": "TYPE", "className": "nws-tbl-type" },
-			{ "data": "TEASER",  "className": "nws-tbl-desc" },
-			{ "data": "LOCATION",  "visible": false },
-			{ "data": "AUDIENCE",  "visible": false },
-			{ "data": "SUBJECT", "visible": false },
-			{ "data": "MINISTER", "visible": false }
-		]}'>
-			<thead class="wb-inv">
-			<tr>
-				<th>Title</th>
-				<th>Publication date</th>
-				<th>Department</th>
-				<th>News Type</th>
-				<th>Summary</th>
-				<th>Location</th>
-				<th>For</th>
-				<th>Subject</th>
-				<th>Minister</th>
-			</tr>
-			</thead>
-		</table>
+		<p>To access all articles, please visit the <a href="https://www.canada.ca/en/news/advanced-news-search.html">advanced search</a>.</p>
```

Comment:
I recommend to point directly to the search result page, instead of sending the user to make a search. That will remove 1 step for the user in order to get all the news article.

```suggestion
		<p>To access all articles, please visit the <a href="https://www.canada.ca/en/news/advanced-news-search/news-results.html?start=&end=">advanced search</a>.</p>
```